docs/modules/ROOT/pages/guides/erc20-supply.adoc

```diff
@@ -184,7 +146,6 @@ mod MyToken {
 }
 ----
 
-In the constructor, we pass the owner address to set the owner of the `MyToken` contract.
```

Comment:
i don't think we should remove this, especially if we're not showing it in the code